src/components/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+function RegistrationForm(props) {
+  const [state, setState] = useState({
+    email: "",
+    password: "",
+    confirmPassword: "",
+  });
+  const handleChange = (e) => {
+    const { id, value } = e.target;
+    setState((prevState) => ({
+      ...prevState,
+      [id]: value,
+    }));
+  };
+  const handleSubmitClick = (e) => {
+    e.preventDefault();
+    if (state.password === state.confirmPassword) {
+      console.log("email: ", state.email, "password: ", state.password);
+    } else {
+      console.log("Passwords do not match!");
+    }
+  };
+  return (
+    <div className="card col-12 col-lg-4 login-card mt-2 hv-center">
+      <form>
+        <div className="form-group text-left">
+          <label htmlFor="exampleInputEmail1">Email Address</label>
+          <input
+            type="email"
+            className="form-control"
+            id="email"
+            placeholder="Enter email"
+            value={state.email}
+            onChange={handleChange}
+          ></input>
+        </div>
+        <div className="form-group text-left">
+          <label htmlFor="exampleInputPassword1">Password</label>
+          <input
+            type="password"
+            className="form-control"
+            id="password"
+            placeholder="Password"
+            value={state.password}
```

Comment:
Can we use destruct to read values from state?

Reply:
Do you mean like this for example?

class MyComponent extends Component {
   constructor(props) {
     super(props)

     this.state = {
       foo: 'foo',
       bar: 'bar'
     }
   }

   render() {
     const { foo, bar } = this.state

     return <Text>{foo}{bar}</Text>
   }
}

Reply:
I guess we can use this in a class component, not in a function component.

Reply:
not really, that example is about class component and "state" in class component means "state" . in here your state is just a name that is using "useState". I shared in another comment that you should first change that state name to something else like " formValues " to avoid this confusion. 

The comment I shared here related to destructure is an ES6 concept that you can de structure any javascript object ( its not even related to React itself its more generic ). It is a common pattern that helps readability so for ANY given like:

const myObj = { name: 'abbas', phone: "123-123-123" }

you can read from that object like this:
myObj.name

or this 
const { name }  = myObj;

the second way is the common ES6 way please read more about it here. since there are MORE usages and scenarios for this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
Sure will do, thanks.

Reply:
Hi Arman, I couldn't get your point. I've read the article but still couldn't understand what should be changed.